sdk/src/main/java/io/opentdf/platform/sdk/KeyType.java

```diff
@@ -0,0 +1,41 @@
+package io.opentdf.platform.sdk;
+
+public enum KeyType {
+    RSA2048Key("rsa:2048"),
+    EC256Key("ec:secp256r1"),
+    EC384Key("ec:secp384r1"),
+    EC521Key("ec:secp521r1");
+
+    private final String keyType;
+
+    KeyType(String keyType) {
+        this.keyType = keyType;
+    }
+
+    @Override
+    public String toString() {
+        return keyType;
+    }
+
```

Comment:
```suggestion
    public boolean isEc() {
        return this != RSA2048Key;
    }
```

This is the same as the logic that's elsewhere but it's a tiny bit clearer, maybe?